### PR TITLE
APP-116 Update build system notification demo

### PR DESCRIPTION
**Pull request by Dubland**
/cc @rasendubi @forGGe

### CODE REVIEW

File: notificationdemo/source/c/build.sh [hunk 1]
```diff
@@ -68,32 +68,20 @@ function build_thirdparty {
...
+    cd $PROJECT_HOME
+    mkdir -p "$PROJECT_HOME/$BUILD_DIR"
+    cd $BUILD_DIR
+    cmake -DCMAKE_BUILD_TYPE=Debug \
+          -DKAA_WITHOUT_CONFIGURATION=1 \
+          -DKAA_WITHOUT_EVENTS=1 \
+          -DKAA_WITHOUT_LOGGING=1 \
+          -DKAA_MAX_LOG_LEVEL=3 \
+          -DAPP_NAME=$APP_NAME \
+	      -DKAA_PLATFORM=$KAA_ARCH \
```
Comment (rasendubi) [inline notificationdemo/source/c/build.sh:83]: alignment

File: notificationdemo/source/c/build.sh [hunk 2]
```diff
@@ -68,32 +68,20 @@ function build_thirdparty {
...
-    cd $PROJECT_HOME
 }
 
 function build_app {
-    cd $PROJECT_HOME &&
-    mkdir -p "$PROJECT_HOME/$BUILD_DIR" &&
-    cp "$KAA_LIB_PATH/$BUILD_DIR/"libkaa* "$PROJECT_HOME/$BUILD_DIR/" &&
-    cd $BUILD_DIR &&
-    cmake -DAPP_NAME=$APP_NAME -DKAA_PLATFORM=$KAA_ARCH $KAA_TOOLCHAIN_PATH_SDK ..
+    cd $PROJECT_HOME
```
Comment (rasendubi) [inline notificationdemo/source/c/build.sh:74]: remove line

File: notificationdemo/source/c/build.sh [hunk 3]
```diff
@@ -68,32 +68,20 @@ function build_thirdparty {
...
+    mkdir -p "$PROJECT_HOME/$BUILD_DIR"
```
Comment (rasendubi) [inline notificationdemo/source/c/build.sh:75]: `mkdir -p "$BUILD_DIR"`

File: notificationdemo/source/c/build.sh [hunk 4]
```diff
@@ -68,32 +68,20 @@ function build_thirdparty {
...
+    cd $BUILD_DIR
+    cmake -DCMAKE_BUILD_TYPE=Debug \
+          -DKAA_WITHOUT_CONFIGURATION=1 \
+          -DKAA_WITHOUT_EVENTS=1 \
+          -DKAA_WITHOUT_LOGGING=1 \
```
Comment (rasendubi) [inline notificationdemo/source/c/build.sh:80]: `KAA_WITHOUT_XXX` flags were renames. Use `-DKAA_WITH_EXTENSION_XXX=OFF`. See https://github.com/kaaproject/kaa/blob/develop/client/client-multi/client-c/CMakeLists.txt#L26

**Comment by Dubland** [comment]
This pull request replaced to APP-63

**Pull request closed without merging**